fiori/app/common.cds

```diff
@@ -71,34 +71,36 @@ annotate my.Books with {
 
 ////////////////////////////////////////////////////////////////////////////
 //
-//	Computed Fields for Tree Tables 
+//	Computed Fields for Tree Tables
 //
 //  DISCLAIMER: The below are an alpha version implementation and will change in final release !!!
 //
 aspect Hierarchy {
-  LimitedDescendantCount : Integer64 = null;
-  DistanceFromRoot       : Integer64 = null;
+  LimitedDescendantCount : Int16 = null;
+  LimitedRank            : Int16 = null;
+  DistanceFromRoot       : Int16 = null;
```

Comment:
Here I also tried changing `Int64` types to `Int16` as discussed → seems to work end-to-end for Fiori clients.